M2/Macaulay2/e/M2FreeAlgebra.hpp

```diff
@@ -166,6 +168,8 @@ class M2FreeAlgebra : public M2FreeAlgebraOrQuotient
   ring_elem makeTerm(const ring_elem a, const_varpower monom) const;
 
   void makeTerm(Poly& result, const ring_elem a, const_varpower monom) const;
+
+  bool is_commutative_ring() const { return false; }
```

Comment:
This is probably unnecessary because it should be inherited from M2FreeAlgebraOrQuotient, no?